.editorconfig

```diff
@@ -2,3 +2,6 @@
 max_line_length = 140
 indent_size = 4
 insert_final_newline = true
+ij_kotlin_allow_trailing_comma = true
+ij_kotlin_allow_trailing_comma_on_call_site = true
+disabled_rules=filename
```

Comment:
If you remove the property, there is just a single offence, claiming I should rename `Utils.kt` to `Resolve.kt`, but I disagree with this rule in general 🤷 If you believe we should adopt all ktlint rules - please let me know, and I'll adjust the code accordingly :)

Reply:
We can leave it be for now.